Cannot find module '@accounts/magic-link'

Using version v2.2.2 seems to have an install issue. The npm install phase will run normally, however when Reaction starts, fails to load. Perhaps a peer-depdendency is missing.

```
Error: Cannot find module '@accounts/magic-link'
  Require stack:
  - /reaction-core/node_modules/@reactioncommerce/api-plugin-authentication/node_modules/@accounts/graphql-api/lib/modules/accounts-magic-link/index.js
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:924:15)
      at Function.Module._load (node:internal/modules/cjs/loader:769:27)
      at Module.require (node:internal/modules/cjs/loader:996:19)
      at require (node:internal/modules/cjs/helpers:92:18)
      at Object.<anonymous> (/reaction-core/node_modules/@reactioncommerce/api-plugin-authentication/node_modules/@accounts/graphql-api/lib/modules/accounts-magic-link/index.js:7:22)
      at Module._compile (node:internal/modules/cjs/loader:1092:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1121:10)
      at Module.load (node:internal/modules/cjs/loader:972:32)
      at Function.Module._load (node:internal/modules/cjs/loader:813:14)
      at Module.require (node:internal/modules/cjs/loader:996:19)
```

--- a/package.json
+++ b/package.json
@@ -26,6 +26,7 @@
   "sideEffects": false,
   "dependencies": {
     "@accounts/graphql-api": "^0.33.0",
+    "@accounts/magic-link": "^0.1.1",
     "@accounts/mongo": "^0.33.0",
     "@accounts/password": "^0.32.0",
     "@accounts/server": "^0.33.0",

